tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -117,6 +117,16 @@ type CompilerAssert private () =
 
 </Project>"""
 
+    static let directoryBuildProps = """
+<Project>
+  <PropertyGroup>
+    <DisableCompilerRedirection>true</DisableCompilerRedirection>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+</Project>
+"""
```

Comment:
Are you trying to prevent _all_ directory crawling?  If so, this file should just be an empty `<Project />` element and a similarly emtpy `Directory.Build.targets` should be present, too.